### ISSUE TITLE
AMD module adapter did not define

### ISSUE THREAD

**Issue by ericcarino** [stealjs/steal#806]
steal 0.16.29
steal-tools 0.16.6

We updated one of our 3rd party libraries and it's causing Steal to throw an exception and halt execution.

```
steal.js:140 Potentially unhandled rejection [3] TypeError: Error loading "adapter" at http://127.0.0.1:8080/adapter.js
AMD module adapter did not define
    at Loader.loader.instantiate (http://127.0.0.1:8080/node_modules/steal/steal.js:4423:17)
    at Loader.loader.instantiate (http://127.0.0.1:8080/node_modules/steal/steal.js:4744:32)
    at Loader.loader.instantiate (http://127.0.0.1:8080/node_modules/steal/steal.js:6365:30)
    at Loader.loader.instantiate (http://127.0.0.1:8080/node_modules/steal/steal.js:5565:56)
    at Loader.loader.instantiate (http://127.0.0.1:8080/node_modules/steal/steal.js:5690:28)
    at http://127.0.0.1:8080/node_modules/steal/steal.js:1548:33
    at tryCatchReject (http://127.0.0.1:8080/node_modules/steal/steal.js:1183:30)
    at runContinuation1 (http://127.0.0.1:8080/node_modules/steal/steal.js:1142:4)
    at Fulfilled.when (http://127.0.0.1:8080/node_modules/steal/steal.js:930:4)
    at ContinuationTask.run (http://127.0.0.1:8080/node_modules/steal/steal.js:1040:24)
```

What's extremely odd is if I delete the following **commented out** block of code from the 3rd party library, everything works fine???

``` 
//    define([], function() {
//       return {
//         RTCPeerConnection: window.RTCPeerConnection,
//         RTCIceCandidate: window.RTCIceCandidate,
//         RTCSessionDescription: window.RTCSessionDescription,
//         getUserMedia: getUserMedia,
//         attachMediaStream: attachMediaStream,
//         reattachMediaStream: reattachMediaStream,
//         webrtcDetectedBrowser: webrtcDetectedBrowser,
//         webrtcDetectedVersion: webrtcDetectedVersion,
//         webrtcMinimumVersion: webrtcMinimumVersion,
//         webrtcTesting: webrtcTesting,
//         webrtcUtils: webrtcUtils
//         requestUserMedia: not exposed on purpose.
//         trace: not exposed on purpose.
//       };
//     });
```

You should be able to reproduce by copy/pasting that block of code into any of your imported modules.

**Comment by pYr0x** [comment]
Can you try it with the steal 1.0 beta? I thought we improve the amd detection .. But I am but sure

**Comment by ericcarino** [comment]
Thanks!  Yes, the issue appears to be fixed in 1.0 beta!
